Script/material.dat

```diff
@@ -1827,6 +1873,44 @@ liquid
     AttachedGod = CRUENTUS;
     Effect = EFFECT_MUSTARD_GAS_LIQUID;
   }
+
+  Config OMMEL_BLOOD;
+  {
+    StrengthValue = 1;
+    NutritionValue = 250;
+    Color = rgb16(180, 20, 20);
+    NameStem = "ommel blood";
+    Alpha = 175;
+    Flexibility = 40;
+    AttachedGod = CRUENTUS;
+    Acidicity = 5000:
+    RustModifier = 5000;
+    ConsumeWisdomLimit = 1;
+    CategoryFlags = IS_BLOOD;
+    InteractionFlags = Base|AFFECT_INSIDE;
+    StepInWisdomLimit = 1;
+    NameFlags = Base|USE_AN;
+    PriceModifier = 5000;
+    CommonFlags = Base|IS_VALUABLE&~CAN_BE_MIRRORED;
+    ConsumeEndMessage = CEM_OMMEL;
+    HitMessage = HM_OMMEL;
+    BodyFlags = Base|IS_WARM;
+  }
+
+  Config LIQUID_DARKNESS;
+  {
+    StrengthValue = 1;
+    Density = 10;
+    NutritionValue = 1;
+    RustModifier = 0;
+    Color = rgb16(0, 0, 0);
+    Emitation = rgb24(35, 35, 35);
+    NameStem = "liquid darkness";
+    Effect = EFFECT_DARKNESS;
```

Comment:
Decrease mental attributes. As mental attributes influence monster AI (AFAIK, I know for sure high-Wis monsters are more clever about avoiding harmful gasses and puddles), this can be used to make monsters more dumb about walking into your traps, grenade gasses or acid rain.